drivers/sensor/adxl375/adxl375.h

```diff
@@ -0,0 +1,348 @@
+/*
+ * Copyright (c) 2023 Aaron Chan
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef ZEPHYR_DRIVERS_SENSOR_ADXL375_adxl375_H_
+#define ZEPHYR_DRIVERS_SENSOR_ADXL375_adxl375_H_
+
+#include <zephyr/drivers/sensor.h>
+#include <zephyr/types.h>
+#include <zephyr/device.h>
+#include <zephyr/drivers/gpio.h>
+#include <zephyr/kernel.h>
+#include <zephyr/sys/util.h>
+
+#if DT_ANY_INST_ON_BUS_STATUS_OKAY(spi)
+#include <zephyr/drivers/spi.h>
+#endif /* DT_ANY_INST_ON_BUS_STATUS_OKAY(spi) */
+
+#if DT_ANY_INST_ON_BUS_STATUS_OKAY(i2c)
+#include <zephyr/drivers/i2c.h>
+#endif /* DT_ANY_INST_ON_BUS_STATUS_OKAY(i2c) */
+
+/*
+ * ADXL375 registers definition
+ */
+#define ADXL375_DEVID            0x00u /* Analog Devices accelerometer ID */
```

Comment:
i'd properly align all the macro values. improves readability.

Reply:
If you're talking about having it one space apart for each, I don't think I have power over this. I clang-formatted every file to meet the CI checks and this is how the formatter wanted to do it